.circleci/config.yml

```diff
@@ -12,6 +12,7 @@ jobs:
       - checkout
       - prodsec/security_scans:
           mode: auto
+          open-source-scan: disabled
```

Comment:
temporary, until the issue described [here](https://snyk.slack.com/archives/C073HFTPLSF/p1728301821675079) and [here](https://snyk.slack.com/archives/C069RBK010B/p1728292347080629) is fixed